Sources/AblyChat/Room.swift

```diff
@@ -50,8 +50,9 @@ internal actor DefaultRoom: Room {
         fatalError("Not yet implemented")
     }
 
-    public nonisolated var status: any RoomStatus {
-        fatalError("Not yet implemented")
+    private let _status = DefaultRoomStatus()
```

Comment:
I guess you mean declarations of stored properties (which makes sense, because then you can see how the initializer initializes them)? I’ve moved the `_status` declaration to the top.

Reply:
>I guess you mean declarations of stored properties

Yes. Thanks 👍